hazelcast/src/main/java/com/hazelcast/spi/properties/ClusterProperty.java

```diff
@@ -1626,6 +1629,140 @@ private int getWhenNoSSLDetected() {
     public static final HazelcastProperty MOBY_NAMING_ENABLED
             = new HazelcastProperty("hazelcast.member.naming.moby.enabled", true);
 
+    /*
+     * JET PROPERTIES
+     */
+    /**
+     * Jet will periodically check for new jobs to start and perform cleanup of
+     * unused resources. This property configures how often this check and
+     * cleanup will be done. Value is in milliseconds.
+     *
+     * @since Jet 3.2
+     */
+    public static final HazelcastProperty JOB_SCAN_PERIOD
+            = new HazelcastProperty("jet.job.scan.period", SECONDS.toMillis(5), MILLISECONDS);
```

Comment:
The issue mentions the rest of the properties is prefixed with `hazelcast.`.

We should prefix all these, e.g. `hazelcast.jet.job.scan.period`- to keep it consistent and provide the original non-prefixed key as deprecated name, see `com.hazelcast.spi.properties.HazelcastProperty#setDeprecatedName`.